cmd/gossamer/config.go

```diff
@@ -26,24 +24,22 @@ import (
 
 var (
 	// DefaultCfg is the default configuration for the node.
-	DefaultCfg                = dot.GssmrConfig
-	defaultGssmrConfigPath    = "./chain/gssmr/config.toml"
-	defaultKusamaConfigPath   = "./chain/kusama/config.toml"
-	defaultPolkadotConfigPath = "./chain/polkadot/config.toml"
-	defaultDevConfigPath      = "./chain/dev/config.toml"
-
-	gossamerName = "gssmr"
-	kusamaName   = "kusama"
-	polkadotName = "polkadot"
-	devName      = "dev"
+	DefaultCfg                  = dot.WestendDevConfig
+	defaultKusamaConfigPath     = "./chain/kusama/config.toml"
+	defaultPolkadotConfigPath   = "./chain/polkadot/config.toml"
+	defaultWestendDevConfigPath = "./chain/westen-dev/config.toml"
+
+	kusamaName     = "kusama"
+	polkadotName   = "polkadot"
+	westendDevName = "westend-dev"
 )
 
 // loadConfigFile loads a default config file if --chain is specified, a specific
 // config if --config is specified, or the default gossamer config otherwise.
 func loadConfigFile(ctx *cli.Context, cfg *ctoml.Config) (err error) {
 	cfgPath := ctx.GlobalString(ConfigFlag.Name)
 	if cfgPath == "" {
-		return loadConfig(cfg, defaultGssmrConfigPath)
+		return loadConfig(cfg, defaultPolkadotConfigPath)
```

Comment:
`DefaultCfg = dot.WestendDevConfig`

So, default config is westend dev config, but if while loading config file path is not provided you load it from `defaultPolkadotConfigPath`, why?

`return loadConfig(cfg, defaultPolkadotConfigPath)`

Personally, I feel like our default config should be whatever immediate chain that we are trying to support. So, `westend` instead of westend dev